webapp/src/components/systemConsole/surveyDateTime/surveyDateTime.tsx

```diff
@@ -1,86 +1,61 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {format, parse} from 'date-fns';
+import {format} from 'date-fns';
+import {formatInTimeZone} from 'date-fns-tz';
 import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import utils from 'utils/utils';
 
 import SurveyDateSelector from 'components/surveyDateSelector/surveyDateSelector';
 import SurveyTimeSelector from 'components/surveyTimeSelector/surveyTimeSelector';
 import type {CustomSettingChildComponentProp} from 'components/systemConsole/index';
 
-import type {DateTimeConfig} from 'types/plugin';
-
 import './style.scss';
-
-const DEFAULT_SURVEY_TIME = '00:00';
+import type {DateTimeConfig} from 'types/plugin';
```

Comment:
Surprised this is not failing on CI. This import should go before the unnamed style import.